lib/mayaUsd/ufe/UsdTransform3d.cpp

```diff
@@ -92,107 +92,107 @@ void UsdTransform3d::setItem(const UsdSceneItem::Ptr& item)
+        {
+            x = v[0]; y = v[1]; z = v[2];
+        }
+    }
+    return Ufe::Vector3d(x, y, z);
 }
 
 #if UFE_PREVIEW_VERSION_NUM >= 2013
 Ufe::Vector3d UsdTransform3d::rotation() const
 {
-	double x{0}, y{0}, z{0};
-	const TfToken rotXYZ("xformOp:rotateXYZ");
-	if (fPrim.HasAttribute(rotXYZ))
-	{
-		// Initially, attribute can be created, but have no value.
-		GfVec3f v;
-		if (fPrim.GetAttribute(rotXYZ).Get<GfVec3f>(&v,getTime(path())))
-		{
-			x = v[0]; y = v[1]; z = v[2];
-		}
-	}
-	return Ufe::Vector3d(x, y, z);
+    double x{0}, y{0}, z{0};
+    const TfToken rotXYZ("xformOp:rotateXYZ");
+    if (fPrim.HasAttribute(rotXYZ))
+    {
+        // Initially, attribute can be created, but have no value.
+        GfVec3f v;
+        if (fPrim.GetAttribute(rotXYZ).Get<GfVec3f>(&v,getTime(path())))
+        {
+            x = v[0]; y = v[1]; z = v[2];
+        }
+    }
+    return Ufe::Vector3d(x, y, z);
 }
 
 Ufe::Vector3d UsdTransform3d::scale() const
 {
-	double x{0}, y{0}, z{0};
-	const TfToken scaleTok("xformOp:scale");
-	if (fPrim.HasAttribute(scaleTok))
-	{
-		// Initially, attribute can be created, but have no value.
-		GfVec3f v;
-		if (fPrim.GetAttribute(scaleTok).Get<GfVec3f>(&v,getTime(path())))
-		{
-			x = v[0]; y = v[1]; z = v[2];
-		}
-	}
-	return Ufe::Vector3d(x, y, z);
+    double x{0}, y{0}, z{0};
+    const TfToken scaleTok("xformOp:scale");
+    if (fPrim.HasAttribute(scaleTok))
+    {
+        // Initially, attribute can be created, but have no value.
+        GfVec3f v;
+        if (fPrim.GetAttribute(scaleTok).Get<GfVec3f>(&v,getTime(path())))
+        {
+            x = v[0]; y = v[1]; z = v[2];
+        }
+    }
+    return Ufe::Vector3d(x, y, z);
 }
 
 Ufe::RotateUndoableCommand::Ptr UsdTransform3d::rotateCmd(double x, double y, double z)
 {
-	return UsdRotateUndoableCommand::create(fItem, x, y, z);
+    return UsdRotateUndoableCommand::create(path(), x, y, z);
 }
 #endif
 
 void UsdTransform3d::rotate(double x, double y, double z)
 {
-	rotateOp(fPrim, fItem->path(), x, y, z);
+    rotateOp(fPrim, fItem->path(), x, y, z);
 }
 
 #if UFE_PREVIEW_VERSION_NUM >= 2013
 Ufe::ScaleUndoableCommand::Ptr UsdTransform3d::scaleCmd(double x, double y, double z)
 {
-	return UsdScaleUndoableCommand::create(fItem, x, y, z);
+    return UsdScaleUndoableCommand::create(path(), x, y, z);
 }
 
 #else
 
 Ufe::TranslateUndoableCommand::Ptr UsdTransform3d::translateCmd()
 {
-	return UsdTranslateUndoableCommand::create(fItem, 0, 0, 0);
+    return UsdTranslateUndoableCommand::create(path(), 0, 0, 0);
 }
 
 Ufe::RotateUndoableCommand::Ptr UsdTransform3d::rotateCmd()
 {
-	return UsdRotateUndoableCommand::create(fItem, 0, 0, 0);
+    return UsdRotateUndoableCommand::create(path(), 0, 0, 0);
 }
 
 Ufe::ScaleUndoableCommand::Ptr UsdTransform3d::scaleCmd()
 {
-	return UsdScaleUndoableCommand::create(fItem, 1, 1, 1);
+    return UsdScaleUndoableCommand::create(path(), 1, 1, 1);
 }
```

Comment:
@ppt-adsk  please ignore the whitespace changes. The main change here is that we are now passing the pass directly to 
Translate/Rotate/Scale/RotatePivotTranslate commands.